Core/src/org/sleuthkit/autopsy/communications/VisualizationPanel.java

```diff
@@ -505,6 +522,16 @@ public void actionPerformed(ActionEvent evt) {
 
         jSeparator2.setOrientation(SwingConstants.VERTICAL);
 
+        snapshotButton.setIcon(new ImageIcon(getClass().getResource("/org/sleuthkit/autopsy/report/images/image.png"))); // NOI18N
+        snapshotButton.setText(NbBundle.getMessage(VisualizationPanel.class, "VisualizationPanel.snapshotButton.text_1")); // NOI18N
+        snapshotButton.addActionListener(new ActionListener() {
```

Comment:
I would use a method reference here.

Reply:
Generally I would too, but this code is generated from the ide and is consistent with the rest of the code.  I am reluctant to change just this one.